cli/cmd/setup.go

```diff
@@ -17,21 +17,52 @@ package cmd
 import (
 	"fmt"
 
+	"github.com/kosyfrances/rundeck-zabbix/lib"
+	"github.com/prometheus/common/log"
 	"github.com/spf13/cobra"
 )
 
 // setupCmd represents the setup command
 var setupCmd = &cobra.Command{
 	Use:   "setup",
-	Short: "A brief description of your command",
-	Long: `A longer description that spans multiple lines and likely contains examples
-and usage of using your command. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Short: "initialise tool configuration",
+	Long:  `initialise tool configuration`,
```

Comment:
Since `Long and `Short` text are the same, what do you think about leaving out `Long`?

Reply:
I will update this on a new pr